hasura/hasura-metadata/tables.yaml

```diff
@@ -48,6 +48,34 @@
           table:
             name: deposits
             schema: public
+    - name: group_members
+      using:
+        foreign_key_constraint_on:
+          column: account_id
```

Comment:
No need to have a constrain on group_id column?

Reply:
this is a relation under the account table. There is constrain on group_id in the group table.